server/startup.coffee

```diff
@@ -94,6 +96,26 @@ Meteor.startup ->
               id: "userSpecifiedDisease:#{incident.disease}"
               text: "Other Disease: #{incident.disease}"
 
+  promedFeed = Feeds.findOne(url: $regex: /promedmail.org/)
```

Comment:
Let's replace these regex's with an exact url for the automatically added ProMED feed. If we add ProMED regional feeds this type of matching could pick up the wrong feed.